### DISCUSSION TITLE
GHelper Anime Matrix Rog Logo

### DISCUSSION THREAD

**Discussion by BogdanT07** [seerge/g-helper#1952] [Q&A]
Hi, I just tested the GHelper on a M16 zephyrus and all good! Is really NiCE!
Just wanted to modify the Rog Logo from the app with some other gif.
Looked into the code but couldnt find MatrixLogo.

Can you help me and tell me where from is getting that gif for Rog Logo? Thanks!

LE: I know there's an option for custom gif, but the Rog Logo remains set even on reboot  This and BinnaryBanner.  So when i go into Ubuntu, these 2 remain active. But the custom gif wont. So I was wondering if I can load a specific gif just as BinnaryBanner and RogLogo were loaded (i hope they arent predefined by Asus or anything)

**Comment by IceStormNG** [comment]
The ROG Logo is an in-built animation. The matrix has a few animations that are embedded into the firmware and these can also run when GHelper/AC are not active. There are only 2-3 of them that work that way. They cannot be edited in any way.

**Reply by BogdanT07** [reply]
Oh i see, i was thinking to customize one for the "offline" mode, but there is no way so I have to quit it :( 
Thanks for the answer. The app kicks asses actually, way way better and convenient than the armory stuff. Keep it up!
